shraga_common/app/routers/analytics_api.py

```diff
@@ -13,8 +13,8 @@ async def get_analytics(r: AnalyticsRequest) -> dict:
     return await analytics_service.get_analytics(r)
 
 
-@router.get("/chat-history", response_model=List[Chat])
-async def get_chat_history(
+@router.get("/chat-dialogs", response_model=List[ChatMessage])
```

Comment:
Please add docs about this function.
i.e. that it returns pairs of questions and answers.